website/static/js/notificationsTreebeard.js

```diff
@@ -173,9 +173,18 @@ function ProjectNotifications(data) {
                                     subscribe(item, ev.target.value);
                                 }},
                                 [
-                                    item.data.event.title !== 'comment_replies' ? m('option', {value: 'none', selected : item.data.event.notificationType === 'none' ? 'selected': ''}, 'Never') : null,
-                                    m('option', {value: 'email_transactional', selected : item.data.event.notificationType === 'email_transactional' ? 'selected': ''}, 'Instantly'),
-                                    m('option', {value: 'email_digest', selected : item.data.event.notificationType === 'email_digest' ? 'selected': ''}, 'Daily')
+                                    item.data.event.title !== 'comment_replies' && !~item.data.event.title.indexOf('mentions') ?
+                                        m('option', {value: 'none', selected : item.data.event.notificationType === 'none' ? 'selected': ''}, 'Never') :
+                                        null,
+                                    ~item.data.event.title.indexOf('mentions') ?
```

Comment:
Sorry, I oversimplified that answer. A better explanation:

>
The obscure '~' operator in javascript performs the operation "value * -1 - 1", e.g. ~-2 === 1. The only use case I have ever had for this is in converting the "not found" -1 from ".indexOf()" into "0" (a falsey value in javascript), follow through and see it will convert an index found at position "0" to "-1", a truthy value.
>

Source: [stackoverflow](http://stackoverflow.com/questions/20906471/best-way-to-convert-result-of-indexof-to-a-boolean-using-javascript-or-jquery/20906514)